src/lib/components/ProjectsPanel/components/ProjectsPanel.tsx

```diff
@@ -0,0 +1,97 @@
+import styles from "./ProjectsPanel.module.css";
+import classNames from "classnames";
+import { ALL_PROJECTS, PROJECT_MAP } from "../constants";
+import { Styles, Project, Theme, ToggleStyles } from "../types";
+import { Projects } from "./Projects";
+import { Modal, useModal } from "./Modal";
+
+const defaultTheme = {
+  background: "black",
+  color: "white",
+};
+
+const defaultStyle = { bottom: "16px", left: "16px", zIndex: 1000 };
+
+export interface Props {
+  activeProjectId?: Project["id"];
+  projects?: Project[];
+  theme?: Theme;
+  isDarkIcons?: boolean;
```

Comment:
Do we really need the word `is` in React Props?)

Reply:
I mark with `is` boolean props)